### PR TITLE
Added and stylized recent-works and about-myself sections

### PR DESCRIPTION
**Pull request by aghtetnyein**
I added the Recent Works section and About Myself sections and stylized for mobile view.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,5 +53,143 @@ <h5 class="small-intro">I’am a Software Developer</h5>
...
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="tags">
+          <div class="tag-item">css</div>
+          <div class="tag-item">html</div>
+          <div class="tag-item">bootstrap</div>
+          <div class="tag-item">Ruby</div>
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. Kindly work on it. :+1:

File: index.html [hunk 2]
```diff
@@ -53,5 +53,143 @@ <h5 class="small-intro">I’am a Software Developer</h5>
...
+              <div class="tag-item">Ruby</div>
+            </div>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!-- about me -->
+    <section id="about-me">
```
Comment (mahbubul14) [inline index.html:140]: - [x] Your page has a horizontal scrollbar. Kindly fix the issue. :+1: 

https://user-images.githubusercontent.com/70483835/185478806-2ecc00e6-dcb9-4b64-aab2-592a0cd13e65.mp4

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @aghtetnyein ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [x] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @aghtetnyein 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**